pyteal/ast/scratch.py

```diff
@@ -156,7 +226,7 @@ def __init__(self, slot: ScratchSlot):
         self.slot = slot
 
     def __str__(self):
-        return "(StackStore {})".format(self.slot)
+        return "(ScratchStackStore {})".format(self.slot)
```

Comment:
`ScratchStackStore` is unique and potentially destructive, so it should be easy to tell it apart from a normal store in my opinion. If it where merged with `ScratchStore`, it might be too easy to accidentally enable it when you didn't mean to.

(comment about the actual line change): If the other `__str__` methods are `Store` and `Load`, I think this should remain `StackStore`

Reply:
It's still not entirely clear to me what our philosophy for `__str__()` is in PyTeal. I found these methods useful when debugging, and therefore it was useful for me to be able to distinguish between a `StackStore` and `ScratchStackStore` before it was headed for TEAL assembly. However, there might be a more important purpose whereby it's important to keep the printouts the same. I'll add this to our talking points today.

Reply:
👍